test/functional/test_framework/key.py

```diff
@@ -368,15 +379,18 @@ def get_pubkey(self):
         ret.compressed = self.compressed
         return ret
 
-    def sign_ecdsa(self, msg, low_s=True):
+    def sign_ecdsa(self, msg, low_s=True, rfc6979=False):
```

Comment:
Why was this added? It isn't used anywhere in the changes.

Reply:
It is, indirectly through the "deterministic" flag in the feature_taproot's context, which on its turn is used in the test vector scenario (to make sure the fully signed transaction is exactly the same every run).

Reply:
My mistake. I see it now. Thanks.